subsquid/src/pabloProcessor.ts

```diff
@@ -163,13 +162,12 @@ interface LiquidityAddedEvent {
 function getLiquidityAddedEvent(
   event: PabloLiquidityAddedEvent
 ): LiquidityAddedEvent {
-  if (event.isV2300) {
-    const { who, poolId, baseAmount, quoteAmount, mintedLp } = event.asV2300;
-    return { who, poolId, baseAmount, quoteAmount, mintedLp };
-  } else {
-    const { who, poolId, baseAmount, quoteAmount, mintedLp } = event.asLatest;
+  if (event.isV2400) {
+    const { who, poolId, baseAmount, quoteAmount, mintedLp } = event.asV2400;
     return { who, poolId, baseAmount, quoteAmount, mintedLp };
   }
+  const { who, poolId, baseAmount, quoteAmount, mintedLp } = event.asLatest;
+  return { who, poolId, baseAmount, quoteAmount, mintedLp };
```

Comment:
I think in the end we can ignore the restructuring as it doesn't add much value here, so: 
 ```suggestion
 return event.isV2400 ?? event.asLatest;
```

Reply:
This can be used throughout all event getters.